### ISSUE TITLE
Trouble getting Flask-User and SocketIO to work

### ISSUE THREAD

**Issue by dlpersson** [miguelgrinberg/Flask-SocketIO#750]
Hello,

Firstly, thank you for your work on this library. It is excellent

I am attempting to use Flask-User and SocketIO but I cant figure out why the latency spikes to >700ms. The more complex client I am working on at work gets constant HTTP 400s to polling requests, but I can't duplicate that issue with this simple setup. In both environments it is run locally just using whatever is built into Flask.

When the app is first started it is incredibly slow. The ping/pong times hover around 700+ms. An emit takes several seconds to appear. Once socketIO is added all page loads become very slow. I haven't been programming very long and I have no idea where to see what is causing the bottleneck

I created a github repository that shows my issue.
https://github.com/dlpersson/Flask-User-SocketIO-Issue.git

If you clone this you can start the server by running the command "python manage.py runserver" from the directory with the manage.py file.

The socketIO endpoints are in app/views/main_views
The HTML page is in app/templates/main/socketio.html
The App itself is configured in app/__init__.py

Any help or direction would be greatly appreciated

**Comment by miguelgrinberg** [comment]
Do you have eventlet or gevent installed?

**Comment by dlpersson** [comment]
I installed eventlet and had to put the money patch in the create_app method. I eventually got it working when I added socketio.run(app) to the bottom of the create_app method.

The socketio.run works but it seems to pause execution of the script and the app isnt returned to the flask-script manager. It looks like you already answered what is going on in this thread. https://github.com/miguelgrinberg/Flask-SocketIO/issues/279. 

I think I should be all set. Thanks for responding
